lia-core/src/main/java/com/lithium/community/android/model/response/LiSubscriptions.java

```diff
@@ -62,26 +63,26 @@ public void setId(LiBaseModelImpl.LiInt result) {
     }
 
     public LiMessage getLiMessage() {
-        if (targetObject.has("type") && targetObject.get("type").getAsString().equals("message")) {
-            return LiClientManager.getRestClient().getGson().fromJson(targetObject.toString(), LiMessage.class);
+        if (targetObject != null) {
+            return targetObject;
         } else {
             return null;
         }
     }
 
     public LiBoard getLiBoard() {
-        if (targetObject.has("type") && targetObject.get("type").getAsString().equals("board")) {
-            return LiClientManager.getRestClient().getGson().fromJson(targetObject.toString(), LiBoard.class);
-        } else {
-            return null;
-        }
+        return null;//subscriptions will not have board
```

Comment:
Don't know the business scenario but this seems odd as an outsider.. a method always returning null

Reply:
Subscriptions is an exceptional extension of a BaseModel which is returned as API response, and doesn't have a board.

Reply:
can we get rid of this method then?

Reply:
No, its an implementation of an interface.